src/app/shared/branding.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable } from '@angular/core';
+import { Fabric8UIConfig } from '../shared/config/fabric8-ui-config';
+
+import openshiftLogo from '../../assets/images/OpenShift-io_logo.png';
+import fabric8Logo from '../../assets/images/fabric8_logo.png';
+
+@Injectable()
+export class BrandingService {
+  private isFabric8: boolean;
+  private _logo: string;
```

Comment:
Eventually we will want to allow the user to set their own branding on an on-prem version. So these setting could be set from a config file or stored some place else. I don't consider the constructor as the "environment", but I do agree they should be set by their environment. :)